llvm/tools/llvm-profdata/llvm-profdata.cpp

```diff
@@ -914,9 +922,14 @@ static void mergeInstrProfile(const WeightedFileVector &Inputs,
     exitWithError("unknown format is specified");
 
   // TODO: Maybe we should support correlation with mixture of different
-  // correlation modes(w/wo debug-info/object correlation).
-  if (!DebugInfoFilename.empty() && !BinaryFilename.empty())
-    exitWithError("Expected only one of -debug-info, -binary-file");
+  // correlaxtion modes(w/wo debug-info/object correlation).
```

Comment:
nit: I'm presuming `correlaxtion` is a typo?

Reply:
Done.